ts/features/bonus/cgn/store/actions/activation.ts

```diff
@@ -4,6 +4,12 @@ import {
   createStandardAction
 } from "typesafe-actions";
 import { NullType } from "io-ts";
+import { CgnActivationProgressEnum } from "../reducers/activation";
+
+type ActivationStatus = {
+  status: CgnActivationProgressEnum;
+  activation: any; // Replace when API spec is correctly linked and defined
```

Comment:
```suggestion
  activation: any; // FIXME Replace when API spec is correctly linked and defined
```
Same of TODO
![Schermata 2021-02-02 alle 09 26 09](https://user-images.githubusercontent.com/822471/106572451-b853b680-6538-11eb-98fd-d25857c3165f.png)

warning: To avoid to forget `any` from code (it could be very dangerous), could be a solution define a temporary type that represents `activation`?
In this way when you will have the official type, you will remove the temporary one and you know exactly where they have to be replaced. Leaving `any` you have to remember where to replace. It is prone to errors